homeassistant/components/reolink/__init__.py

```diff
@@ -97,8 +99,17 @@ async def async_check_firmware_update() -> str | Literal[False]:
             try:
                 return await host.api.check_new_firmware()
             except ReolinkError as err:
+                if starting:
```

Comment:
Now that you've got this, you can switch the `firmware_coordinator.async_refresh()` below to `async_config_entry_first_refresh()`, I think?

Reply:
Yes you are right, I have switched it back to `async_config_entry_first_refresh` although it should not really matter much.